justfile

```diff
@@ -0,0 +1,46 @@
+set dotenv-load := false
+
+DOCKER_FILE := "-f docker-compose.yml"
+
+
+install:
+    #! /usr/bin/env sh
+    cd openverse-api
+    pipenv install --dev
+    pipenv run pre-commit install
+
+
+lint:
+    #! /usr/bin/env sh
+    cd openverse-api
+    pipenv run pre-commit run --all-files
+
+
+init: up
+    ./load_sample_data.sh
+
+
+healthcheck:
+    curl "localhost:8000/v1/images/?q=honey"
+
+
+test: up
+    docker-compose exec web bash ./test/run_test.sh
+
+
+testlocal:
+    #! /usr/bin/env sh
+    cd openverse-api
+    pipenv run bash ./test/run_test.sh
+
+
+up:
```

Comment:
Since these docker commands (or at least `up`) are dependencies for other recipes, do you mind moving these 3 to the top of the file?